backend/src/main/java/com/example/backend/risk/Risk.java

```diff
@@ -19,9 +19,9 @@ public class Risk {
     private String riskDescription;
     private List<String> riskReductionMeasures;
 
-    private int healthHazard;
-    private int probability;
-    private int frequency;
+    private HealthHazard healthHazard;
+    private Probability probability;
```

Comment:
Be careful when using too many enums. This leads to too many ifs later on. Probability sounds like a double value 0.0 to 1.0. I don't know the exact intention of this field. If this does not apply, please ignore this comment.

Reply:
Hmmm, well, all 3 of these are discrete ints between and including 1 and 4, though they often described with their respective names... So I'm not entirely sure whether I should pick ints or enums. Your suggestion would be using ints, it take it?

Reply:
_Changed_